scripts/validate-release

```diff
@@ -28,7 +28,14 @@ function parse_tag() {
 }
 
 function check_release_branch() {
-    TAG_BRANCH=$(git branch --all -q --contains $GIT_TAG | grep -v origin$ | grep -v "HEAD" | sed -e 's/^[[:space:]]*//')
+    TAG_BRANCH=$(git branch --all -q --contains $GIT_TAG | grep origin | grep -v origin$ | grep -v "HEAD" | sed -e 's/^[[:space:]]*//')
+    if [ "$TAG_BRANCH" == "remotes/origin/master" ]; then
+        K8S_VERSION_GO_MOD=$(grep 'k8s.io/kubernetes v' go.mod | head -n1 | awk '{print $2}')
+        if [ "$MAJOR.$MINOR" == "$K8S_VERSION_GO_MOD" ]; then
```

Comment:
K8S_VERSION_GO_MOD is going to be v1.22.2 or something like that, so it won't compare properly. Maybe try using:
`K8S_VERSION_GO_MOD=$(grep 'k8s.io/kubernetes v' go.mod | awk -F. 'NR==1 {print "1." $3}')`

or compare to $MAJOR.$MINOR.$PATCH